### ISSUE TITLE
Docker Support

### ISSUE THREAD

**Issue by carlosmartinsrodrigues** [Slamtec/sllidar_ros2#35]
Hi has anyone been able to put this running in Docker? 
And having the host running and consuming the data via foxglove?

I'm using A1M8 

Thank you

**Comment by wvergouwenavular** [comment]
Yes, I've been running this node in a docker container on my robots. This is not different than any other ros node, right?
